src/main/java/net/revelc/code/formatter/FormatterMojo.java

```diff
@@ -528,7 +528,7 @@ private List<File> addCollectionFiles(final Resource resource) {
             if (log.isDebugEnabled()) {
                 log.debug(String.format("Skipping non-existing directory %s", newBasedir));
             }
-            return Collections.emptyList();
+            return List.of(Collections.emptyList().toArray(new File[] {}));
```

Comment:
Sorry, but this seems quite insane. Collections.emptyList() was fine. If we want, we can use:

```suggestion
            return List.of();
```

Reply:
Accepting the suggestion here as List.of() is immutable copy of empty.  Will have up shortly.